Sources/FluentKit/Query/Builder/QueryBuilder+Paginate.swift

```diff
@@ -136,10 +136,10 @@ public struct PageRequest: Decodable, Sendable {
     }
 
     var start: Int {
-        (self.page - 1) * self.per
+        (self.page - 1).multipliedReportingOverflow(by: self.per).partialValue
```

Comment:
If we're just going to use the partial value result anyway, don't bother going through `multipliedReportingOverflow(by:)`, just disable the runtime check by requesting modulus arithmetic:
```suggestion
        (self.page - 1) &* self.per
```

Reply:
Nice, didn't know this existed to be honest.